OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1092,6 +1092,15 @@ internal static void UpgradeWeaponRules(ModData modData, int engineVersion, ref
 					}
 				}
 
+				// Rename BurstDelay to BurstDelays
+				if (engineVersion < 20170716)
+				{
+					if (node.Key == "BurstDelay")
+						node.Key = "BurstDelays";
+					if (node.Key == "-BurstDelay")
```

Comment:
It looks unnecessary/wrong to me. Isn't `-Name`... applied only for Traits and not properties?